.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -35,7 +35,7 @@ body:
       label: NuGet package version
       description: Specify the version of Windows App SDK (or Project Reunion) you're using.
       options:
-        - "Windows App SDK 1.3.3: 1.3.230724000"
+        - "Windows App SDK 1.4.0: 1.4.230822000"
```

Comment:
you can remove preview 2

Reply:
And Experimental 1?

Reply:
Experimental 1 has some APIs that aren't present in the stable release.  It should stay.

Reply:
Sounds good, thanks!